### PR TITLE
Update "Get started" documentation with new process for user permissions

### PR DESCRIPTION
**Pull request by theseanything**
This updates the "Get started" documentation to reference the correct process for getting access to GitHub, AWS and other third party services.

Since govuk-user-reviewers now automates the process of apply permissions, joiners no longer need to manually give themselves permissions.

This also refactors the documentation to link to external documentation where possible.

### CODE REVIEW

File: source/manual/get-started.html.md [hunk 1]
```diff
@@ -90,9 +90,8 @@ If you have a YubiKey, you will use `gpg-agent` in place of `ssh-agent`, which r
 
 ## 3. Set up your GitHub account
 
-1. Set up a [GitHub] account (or use your existing personal account).
+1. [Login into your existing GitHub account][github-login] or [create a new GitHub account][github-signup].
 1. [Associate your GitHub account with your GDS email address][associate-email-github], which can be in addition to your personal email address.
```
Comment (sengi) [inline source/manual/get-started.html.md:94]: Do we need a step to tell people to enable 2-factor for their GitHub accounts? Or are we already enforcing that at org level so it'll make them set it up anyway?

File: source/manual/get-started.html.md [hunk 2]
```diff
@@ -102,18 +101,40 @@ If you have a YubiKey, you will use `gpg-agent` in place of `ssh-agent`, which r
...
+[request-aws-user]: https://gds-request-an-aws-account.cloudapps.digital/
+[enable-mfa]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_virtual.html#enable-virt-mfa-for-iam-user
+[yubikey-aws-mfa]: /manual/using-a-yubikey.html#setting-up-aws-mfa
+[aws-cli-auth]: https://docs.aws.amazon.com/cli/latest/userguide/cli-authentication-user.html#cli-authentication-user-get
+
+## 5. Get permissions for AWS, GitHub and other third party services
+
+Permissions to GOV.UK's AWS, GitHub, Fastly, Sentry and Pagerduty accounts are managed by the govuk-user-reviewer repository.
+
+Ask your tech lead to follow these [instructions][] to grant you access.
```
Comment (sengi) [inline source/manual/get-started.html.md:129]: tiny nit: the `[]` is superflous

```suggestion
Ask your tech lead to follow these [instructions] to grant you access.
```

File: source/manual/get-started.html.md [hunk 3]
```diff
@@ -135,11 +156,38 @@ On GOV.UK we use the following command-line tools for AWS and SSH access:
...
+    Successfully initialised gds-cli
+    Enter token for arn:aws:iam::123456789012:mfa/firstname.lastname@digital.cabinet-office.gov.uk: 123456
+    ```
+
+If you have a GDS-issued Yubikey, you can run `gds config yubikey true` in the GDS CLI to set GDS CLI to automatically pull the MFA code from your Yubikey.
+
+You have completed the get started process. You can now use `gds aws` to run generic [aws CLI](https://aws.amazon.com/cli/) commands by prefixing them with `gds aws <role>`. For example:
+
+```shell
+$ gds aws govuk-integration-readonly aws s3 ls
```
Comment (sengi) [inline source/manual/get-started.html.md:184]: nit: omitting the `$` prompt makes the copy button in GitHub more useful (though devdocs doesn't have a copy button yet so it's moot for now, but hey)
```suggestion
gds aws govuk-integration-readonly aws s3 ls
```

File: source/manual/using-a-yubikey.html.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+1. Sign in to the [`gds-users` AWS console][gds-users-aws-signin].
+1. Select the __IAM__ service.
+1. Select __Users__ in the left hand menu and enter your name.
+1. Select the link for your email address.
+1. Select the __Security credentials__ tab.
+1. Select __Manage__, which is next to __Assigned MFA device__.
+1. Specify your email address as the MFA device name
+1. Select "Authenticator app", not "Security Key"
+1. When asked to scan the QR code with your mobile device, open the Yubico Authenticator app and use that to scan the QR code. The MFA code will now be present on your Yubikey.
+1. Configure gds-cli to use the YubiKey:
```
Comment (sengi) [inline source/manual/using-a-yubikey.html.md:21]: optional: for bonus credit, could add a step to [disable the legacy "OTP" app on the Yubikey](https://support.yubico.com/hc/en-us/articles/360013714379-Accidentally-Triggering-OTP-Codes-with-Your-Nano-YubiKey) so you don't paste garbagey strings into Slack and whatnot 😄 (can also do this with `ykman` if that's easier to write instructions for)

**Review by sengi — APPROVED** [review]
Awesome, thanks for updating this!

File: source/manual/get-started.html.md (hunk 1 above)
Comment (theseanything) [reply source/manual/get-started.html.md:94]: Yup enforced at a org level

File: source/manual/get-started.html.md (hunk 3 above)
Comment (theseanything) [reply source/manual/get-started.html.md:184]: Good shout - having the prompts in docs is pet peeve of mine.

**Pull request merged**